CONTRIBUTING.md

```diff
@@ -85,7 +85,9 @@ Run the following command:
 npm run start:api --project_path=<path-to-Kedro-project>
 ```
 
-This command will launch a Kedro-Viz server at [localhost:4142](http://localhost:4142) and serve data from a real Kedro pipeline located at the project path supplied to the command.
+This command will launch a Kedro-Viz server at [localhost:4142](http://localhost:4142) and serve data from a real Kedro pipeline located at the project path supplied to the command. From then on, launching the app locally at [localhost:4141](http://localhost:4141) will pull data from the Kedro-Viz server that is running on port 4142. 
+
+Note: Once the development server is launched at port 4142, the local app will always pull data from that server. To prevent this, you can comment out the proxy setting in `package.json` and restart the dev server at port 4141.
```

Comment:
```suggestion
> **Note**: Once the development server is launched at port 4142, the local app will always pull data from that server. To prevent this, you can comment out the proxy setting in `package.json` and restart the dev server at port 4141.
```

Reply:
Just adding some formatting to your note.

Reply:
Thank you @stichbury ! I'll add that in.

Reply:
on going through the notes again while adding the suggested formatting, I noticed on line 42 of the docs that we use `> _Note:_` to format notes instead of `> **Note**` - my guess is to follow the previous formatting? ( I could also change the formatting within the entire document to follow `> **Note**` instead as well.) @stichbury WDYT?

Reply:
I agree, bold is probably better than italics for this, and I'd change the other one to use bold as well, for consistency's sake